src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1292,31 +1292,33 @@ public FeedbackQuestionAttributes getFeedbackQuestion(String feedbackQuestionId)
     /**
      * Generates summary results (without comments) in CSV format. <br>
      * Preconditions: <br>
-     * * All parameters are non-null. <br>
+     * * All parameters(except questionId) are non-null. <br>
+     * @see FeedbackSessionsLogic#getFeedbackSessionResultsSummaryAsCsv(String, String, String,
+     *      String, String, boolean, boolean)
      */
-    public String getFeedbackSessionResultSummaryAsCsv(String courseId,
-                                                       String feedbackSessionName,
-                                                       String instructorEmail,
-                                                       String filterText,
-                                                       boolean isMissingResponsesShown,
-                                                       boolean isStatsShown)
+    public String getFeedbackSessionResultSummaryAsCsv(
+            String courseId, String feedbackSessionName, String instructorEmail,
+            String filterText, boolean isMissingResponsesShown, boolean isStatsShown, String questionId)
             throws EntityDoesNotExistException, ExceedingRangeException {
 
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, feedbackSessionName);
 
         return feedbackSessionsLogic.getFeedbackSessionResultsSummaryAsCsv(
-                feedbackSessionName, courseId, instructorEmail, filterText, isMissingResponsesShown, isStatsShown);
+                feedbackSessionName, courseId, instructorEmail, questionId,
+                filterText, isMissingResponsesShown, isStatsShown);
     }
 
     /**
      * Generates summary results (without comments) within a section in CSV format. <br>
      * Preconditions: <br>
-     * * All parameters are non-null. <br>
+     * * All parameters(except questionId) are non-null. <br>
+     * @see FeedbackSessionsLogic#getFeedbackSessionResultsSummaryInSectionAsCsv(String, String, String,
+     *      String, String, String, boolean, boolean)
```

Comment:
> Pointing to FeedbackSessionsLogic here is not appropriate because users of Logic API is not supposed to know those lower level classes.

Many methods in this class has this kind of `@see` annotation, most likely because this API is a facade.